README.adoc

```diff
@@ -155,7 +157,14 @@ Revisit the `SystemClient` interface from the previous section:
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[tags=annotations]
 ----
-The `@Dependent` and `@RegisterRestClient` annotations imply that the interface is manageable through Context and Dependency Injection (CDI).
+The `@Dependent` annotation tells the service that the scope of the interface depends on the class that it is injected into.
+By default, interfaces have a scope of `@Dependent` unless there is another scope defined on the interface.
```

Comment:
Remove "there is" and change to..."By default, interfaces have a **`@Dependent` scope** unless **another scope is defined** on the interface."